themes/default/content/blog/tour-pulumi-equinix-provider/index.md

```diff
@@ -0,0 +1,163 @@
+
+<!--more-->
+
+## Introducing the Equinix Provider
+
+The Equinix provider can manage resources for:
+
+- [Equinix Metal](https://www.equinix.com/products/digital-infrastructure-services/equinix-metal), which provides high-performance, bare-metal compute resources.
+- [Equinix Fabric](https://www.equinix.com/products/digital-infrastructure-services/equinix-fabric), which provides software-defined networking that enables low-latency connections between private networks; select service providers like Salesforce, AWS, Azure, and more; and Equinix Metal devices.
+- [Equinix Network Edge](https://www.equinix.com/products/digital-infrastructure-services/network-edge), which enables organizations to deploy virtual network functions (VNFs) (like virtual firewall devices) in a centralized point, greatly simplifying network traffic management for multi- and hybrid cloud scenarios.
+
+Detailed documentation for the Equinix provider can be found in the [Pulumi Registry](https://www.pulumi.com/registry/) along with 130+ (at the time of writing) other providers that can be used to manage cloud and SaaS resources.
+
+{{% notes type="info" %}}
+**Note:** The Equinix provider effectively replaces the now-deprecated [Equinix Metal provider](https://www.pulumi.com/registry/packages/equinix-metal/). To migrate your Pulumi resources to the new provider, use the [`pulumi import`](https://www.pulumi.com/docs/cli/commands/pulumi_import/) command.
+{{% /notes %}}
+
+## Creating a Kubernetes cluster on Equinix Metal
+
+In order to demonstrate the power and utility of Pulumi and the Equinix provider, Equinix Labs has produced a codebase that [creates a Kubernetes cluster on Equinix Metal](https://github.com/equinix-labs/pulumi-equinix-kubernetes-cluster/). The codebase is available in both [TypeScript](https://github.com/equinix-labs/pulumi-equinix-kubernetes-cluster/tree/main/nodejs) and [Python](https://github.com/equinix-labs/pulumi-equinix-kubernetes-cluster/tree/main/python).
+
+The codebase gives an excellent example of one of Pulumi's most compelling features: the ability to manage and orchestrate many different kinds of resources in real programming languages with a single tool. In addition to the Equinix provider which is used to manage the bare metal compute resources, the codebase also uses the following providers:
+
+1. [Cloud-init](https://www.pulumi.com/registry/packages/cloudinit/), to run initialization scripts that install the necessary services to run Kubernetes on the bare metal instances once they are spun up. To get details on what the Cloud-Init scripts are doing on the control plane and worker nodes respectively, see [Kubernetes the Hard Way](https://github.com/kelseyhightower/kubernetes-the-hard-way).
```

Comment:
I'm not sure I'd make a reference to KTHW here, since KTHW doesn't use `kubeadm` at all. No need to block on this, though.

Reply:
Added more nuance here.